config/300-clusterducktype.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: ducktypes.discovery.knative.dev
+  name: clusterducktypes.discovery.knative.dev
```

Comment:
are you planning on creating a ducktype that is namespaced-scope thus you wanted a different name?
I'm wondering if it's the right thing to add info about the scope in the name?

Reply:
I think there are some references in the README to DuckType instead of ClusterDuckType, just double check

Reply:
if we go with ClusterDuckType, update the title of the PR

Reply:
I think we will also add DuckType in a followup when ClusterDuckType is working.

Reply:
>are you planning on creating a ducktype that is namespaced-scope thus you wanted a different name?
I'm wondering if it's the right thing to add info about the scope in the name?
yeah exactly, following the recommended pattern from kubernetes, like RoleBinding vs ClusterRoleBinding

Reply:
ack!